src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java

```diff
@@ -85,7 +85,7 @@ public class VerificationServiceImpl extends CoTopComponent implements Verificat
 	private static String VERIFY_PATH_OUTPUT = CommonFunction.emptyCheckProperty("verify.output.path", "/verify/output");
 	private static String NOTICE_PATH = CommonFunction.emptyCheckProperty("notice.path", "/notice");
 	private static String EXPORT_TEMPLATE_PATH = CommonFunction.emptyCheckProperty("export.template.path", "/template");
-	
+	private static String REVIEW_REPORT_PATH = CommonFunction.emptyCheckProperty("reviewReport.path", "/reviewReport");

```

Comment:
Please add a newline between lines 88-89.

Reply:
Thanks for your detailed review!👍